test/e2e_node/dra_test.go

```diff
@@ -1384,9 +1388,9 @@ func createHealthTestPodAndClaim(ctx context.Context, f *framework.Framework, dr
 	_, err := f.ClientSet.ResourceV1().DeviceClasses().Create(ctx, dc, metav1.CreateOptions{})
 	framework.ExpectNoError(err, "failed to create DeviceClass "+className)
 	ginkgo.DeferCleanup(func() {
-		err := f.ClientSet.ResourceV1().ResourceClaims(f.Namespace.Name).Delete(context.Background(), claimName, metav1.DeleteOptions{})
+		err := f.ClientSet.ResourceV1().DeviceClasses().Delete(context.Background(), className, metav1.DeleteOptions{})
```

Comment:
This works, but it is better to use `func(ctx context.Context)` and that `ctx` here because then Ginkgo can control the duration of shutdown functions.

Reply:
@Jpsassine: this comment still needs to be addressed.